server/dex/dex.go

```diff
@@ -619,6 +622,7 @@ func NewDEX(ctx context.Context, cfg *DexConf) (*DEX, error) {
 		Storage:          storage,
 		AuthManager:      authMgr,
 		BroadcastTimeout: cfg.BroadcastTimeout,
+		TxWaitExpiration: cfg.TxWaitExpiration,
```

Comment:
Does it mean the changes made for auth's `txWaitExpiration` should be reverted?

Reply:
Either reverted or add a second config item just for that.  It would be 'regfeewaitexp` or something, while the swapper's setting would be `swaptxwaitexp`.

I would also add in the docs for the swap one that the value should ideally be less than half of broadcast timeout to allow for more than one retry of the client's request.

Reply:
> Either reverted or add a second config item just for that. It would be regfeewaitexp or something, while the swapper's setting would be swaptxwaitexp.

Reverted. If we really need `regfeewaitexp` to be configurable as we have for swaps,  we can open an issue for it.

Reply:
> If we really need `regfeewaitexp` to be configurable as we have for swaps, we can open an issue for it.

What do you mean?  That's what this setting is for.